vaadin-radio-button-flow-parent/vaadin-radio-button-flow/src/main/java/com/vaadin/flow/component/radiobutton/RadioButtonGroup.java

```diff
@@ -98,6 +107,9 @@
 
     private static <T> T presentationToModel(
             RadioButtonGroup<T> radioButtonGroup, String presentation) {
+        if (radioButtonGroup.keyMapper == null) {
+            return null;
+        }
```

Comment:
Same change as the one for `Select` [here](https://github.com/vaadin/flow-components/pull/4333/files#diff-1c027c9076652ce3a2c641ac7407c10cc3a87846a37ee152c27aa9fe59aad609R226) to make updated unit test pass.